backend/WebApi/Controllers/FoodController.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Contracts.DTOs;
+using Domain.Entities;
+using Microsoft.AspNetCore.Mvc;
+using Services.Interfaces;
+using System.Collections.Generic;
+using System.Net;
+
+namespace WebApi.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/[controller]")]
+    [ApiController]
+    public class FoodController : ControllerBase
+    {
+        private readonly IFoodService _foodService;
+
+        public FoodController(IFoodService foodService)
+        {
+            _foodService = foodService;
+        }
+
+        /// <summary>
+        /// Retrieve all food items
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet]
+        [ProducesResponseType((int)HttpStatusCode.OK, Type = typeof(IEnumerable<Food>))]
+        public IActionResult GetAllFood()
+        {
+            var foods = _foodService.GetAllFood();
+
+            return Ok(foods);
```

Comment:
Maybe also return NotFound if foods is empty?

Reply:
Ok